olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -309,6 +329,10 @@ bool DefaultCacheImpl::RemoveKeysWithPrefix(const std::string& key) {
   RemoveKeysWithPrefixLru(key);
 
   if (mutable_cache_) {
+    auto it = mutable_cache_->NewIterator(leveldb::ReadOptions());
+    for (it->Seek(key); it->Valid() && it->key().starts_with(key); it->Next()) {
+      mutable_cache_data_size -= it->value().size();
+    }
```

Comment:
Instead of doing this here, why dont you make DiskCache::RemoveKeysWithPrefix() return you the size of the deleted data.